src/app/core/models/checkbox.model.ts

```diff
@@ -0,0 +1,19 @@
+export class Checkbox {
+  checkboxString: string; // in the format of - [ ] or - [x]
+
+  constructor(checkboxString: string) {
```

Comment:
I was thinking whether we can just pass in the `description` instead of passing in the `checkbox string` and the `description`

so when we return the checkbox string, it will be something like 

```typescript 
toString(): string {
  return `- ${this.checked ? '[x] ' : '[ ]' } ${this.description} `
}
```

What are your thoughts on this?

Reply:
cc @RuiFengg

Reply:
ohh ya ok it makes more sense to have a separate checked boolean flag and description field in the checkbox model instead of the entire checkbox string, will update it again soon 😄

Reply:
Alright do re-request a review when you are done! :+1: